Header Design

Virtual-Face opened the pull request:
Hi @owocki . Here are 6 designs that I made for the header. Two of them have two versions. I used colors that I think goes well  with the profile page design, but we can change the colors if you like.

V1
![V1](https://user-images.githubusercontent.com/26789429/64487781-071df380-d254-11e9-91b6-7f6b512313c0.jpg)

V2
![V2](https://user-images.githubusercontent.com/26789429/64487726-17819e80-d253-11e9-968f-52c3f96a11da.jpg)

V3
![V3](https://user-images.githubusercontent.com/26789429/64487727-18b2cb80-d253-11e9-9bac-c408513766a4.jpg)

V4
![V4](https://user-images.githubusercontent.com/26789429/64487728-1a7c8f00-d253-11e9-9301-400094d40d06.jpg)

V5A
![V5A](https://user-images.githubusercontent.com/26789429/64487729-1badbc00-d253-11e9-949a-02f485155db8.jpg)

V5B
![V5B](https://user-images.githubusercontent.com/26789429/64487731-1ea8ac80-d253-11e9-98c1-e7bdf74d89e8.jpg)

V6A
![V6A](https://user-images.githubusercontent.com/26789429/64487732-1ea8ac80-d253-11e9-91d9-c5e53efa1ff0.jpg)

V6B
![V6B](https://user-images.githubusercontent.com/26789429/64487733-21a39d00-d253-11e9-843e-ff32d9b2d0e2.jpg)

owocki commented:
thanks @Virtual-Face i really like v5/v6 (both A/B on each)..  love how 'flowing' they feel and how they set the tone for the profile

v1/v2 come off as off-center and feel a bit hacky to me.  v3 and v4 i think we could make work with some color scheme changes that are a bit more muted.

Virtual-Face commented:
I adjusted the V1 and V2, and made some changes too. What do you think?

V1
![V1](https://user-images.githubusercontent.com/26789429/64495788-c229a980-d2b3-11e9-96d4-ff48764d226e.jpg)
V2
![V2](https://user-images.githubusercontent.com/26789429/64495789-c229a980-d2b3-11e9-8815-7f0f208e3a61.jpg)

Changed some colors for V3 and V4, more mute colors. Are they good?

![V3](https://user-images.githubusercontent.com/26789429/64495790-c229a980-d2b3-11e9-82a4-1301e55ed0c1.jpg)
![V4](https://user-images.githubusercontent.com/26789429/64495792-c2c24000-d2b3-11e9-83a4-cd61255c37b7.jpg)

owocki commented:
IMHO the text is distracting on v1; curiuos what others think

for v4; i think this could work if we lose the gitcoin logo and 'build your bounty' (not our tagline anymore!!)

Virtual-Face commented:
I removed the text from V1 and made some changes to V4. What do you think?

![V1](https://user-images.githubusercontent.com/26789429/64511951-9c73c300-d2fa-11e9-9d80-b8ebce9e4163.jpg)

![V4](https://user-images.githubusercontent.com/26789429/64511952-9d0c5980-d2fa-11e9-9b35-cf6d8bdeadb2.jpg)

owocki commented:
LGTM ; curious what @PixelantDesign @willsputra think

Virtual-Face commented:
@owocki  I have a few more ideas for header. Do you want me to work on them?

owocki commented:
hmm.. well we've gone over the original bounty limit already. what are the ideas?

Virtual-Face commented:
I thought it's still open. Looking for next projects. :D
